src/lib/UtapiClient.js

```diff
@@ -131,82 +180,65 @@ export default class UtapiClient {
     * @return {object} this - current instance
     */
     pushMetric(metric, reqUid, params, cb) {
+        assert(methods[metric], `${metric} metric is not handled by Utapi`);
```

Comment:
I was wondering about this, nice that methods that are not yet supported will get an error response.
Does this mean I can uncomment pushmetric calls in my s3 pull requests for bucketCors actions, and just update Utapi/Integration later?